### PR TITLE
Chinese translation file

### PR DESCRIPTION
**Pull request by rayzchen**
由陈智瑞翻译的

### CODE REVIEW

File: eod/translation/languages/zh.json [hunk 1]
```diff
@@ -0,0 +1,209 @@
+{
+  "Name": "Chinese（Simplified）",
```
Comment (Nv7-GitHub) [inline eod/translation/languages/zh.json:2]: Extra space

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]
Seems good! (Just extra space)

**Comment by rayzchen** [comment]
Done

**Review by Nv7-GitHub — APPROVED** [review]
Seems good!

**Pull request merged**